app/Spago/Build.hs

```diff
@@ -57,16 +57,35 @@ repl sourcePaths passthroughArgs = do
   let globs = Packages.getGlobs deps <> defaultSourcePaths <> sourcePaths
   Purs.repl globs passthroughArgs
 
--- | Test the project: compile and run the Test.Main
+-- | Test the project: compile and run "Test.Main"
 --   (or the provided module name) with node
 test :: Maybe Purs.ModuleName -> Maybe Int -> [Purs.SourcePath] -> [Purs.ExtraArg] -> IO ()
-test maybeModuleName maybeLimit paths passthroughArgs = do
+test maybeModuleName successMessage failureMessage maybeLimit paths passthroughArgs =
+  runWithNode (Purs.ModuleName "Test.Main") (Just "Tests succeeded.") "Tests failed: "
+
+-- | Run the project: compile and run "Main"
+--   (or the provided module name) with node
+run :: Maybe Purs.ModuleName -> Maybe Int -> [Purs.SourcePath] -> [Purs.ExtraArg] -> IO ()
+run maybeModuleName successMessage failureMessage maybeLimit paths passthroughArgs =
+  runWithNode (Purs.ModuleName "Main") Nothing "Running failed, exit code: "
+
+-- | Run the project with node: compile and run with the provided ModuleName
+--   (or the default one if that's missing)
+runWithNode :: Purs.ModuleName
+            -> Maybe String
```

Comment:
I'd be fine just with having `pure ()` for success and `Run failed` in case of failure so we could avoid passing this many arguments here, but if we want to then they should be `Text`

Reply:
I'd rather keep the `Text`. Reading the last line of a `test` command as "Tests succeeded." seems like a better UI than just a blank message.